client/components/card/index.jsx

```diff
@@ -105,12 +105,12 @@ let Card = React.createClass( {
 		let fancyTitle;
 		if ( this.props.title ) {
 			fancyTitle = <h2 className="dops-card-title">
```

Comment:
This might be easier to scan if it was wrapped in `()` and the `<h2>` on a new line